cmd/clusterawsadm/gc/gc.go

```diff
@@ -132,7 +131,7 @@ func (c *CmdProcessor) setAnnotationAndPatch(ctx context.Context, annotationValu
 		return fmt.Errorf("creating patch helper: %w", err)
 	}
 
-	annotations.Set(infraObj, expinfrav1.ExternalResourceGCAnnotation, annotationValue)
+	annotations.Set(infraObj, infrav1.ExternalResourceGCAnnotation, annotationValue)
```

Comment:
These changes should be reverted as well.

Reply:
I thought it may be easier to use these constants from api/v1beta2 directly.

Reply:
It would be better to keep it in exp only, to avoid confusion